src/test/java/org/folio/rest/impl/EntitiesCrudTest.java

```diff
@@ -85,6 +92,20 @@ public void testGetById(TestEntities testEntity) throws MalformedURLException {
 
   @ParameterizedTest
   @Order(5)
+  @MethodSource("uniqueKeyConstraint")
+  public void testUniqueKeyConstraint(TestEntities testEntity) throws MalformedURLException {
+    logger.info(String.format("--- mod-orders-storage %s test: Cannot Create Duplicate Entry %s ... ", testEntity.name(), testEntity.name()));
+    JsonObject duplicateEntity = new JsonObject(getSample(testEntity.getSampleFileName()));
+    duplicateEntity.remove("id");
+    Response response = postData(testEntity.getEndpoint(), duplicateEntity.toString());
+    response.then().log().ifValidationFails()
+    .statusCode(400);
+    response.then().body(containsString(ERROR_MESSAGE_UNIQUE_CONSTRAINT));
```

Comment:
RMB provides [ValidationHelper#isDuplicate()](https://github.com/folio-org/raml-module-builder/blob/master/domain-models-runtime/src/main/java/org/folio/rest/tools/utils/ValidationHelper.java#L155) which can be used in such cases

Reply:
changed to use Validation Helper

Reply:
It seems you forgot to push the change to remote branch :)